contrib/devnet/download_snapshot.py

```
@@ -0,0 +1,118 @@
+        sys.exit(1)
+
+    snapshot_path = HOME_DIR / snapshot_filename
+
+    # Download snapshot
+    print(f"\n[2/4] Downloading snapshot to {snapshot_path}...")
+    print("This may take a while depending on your internet connection...")
+    run_command(f'curl "{snapshot_link}" -o "{snapshot_path}"')
+    print("Download complete!")
+
+    # Create temp directory and extract
+    print(f"\n[3/4] Extracting snapshot to temporary location...")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)